test/Combinatorics/EnumerativeCombinatorics/combinations.jl

```diff
@@ -42,4 +42,43 @@
   @test allunique(v)
 
   @test collect(combinations(0, 0)) == [Int[]]
+
+  @testset "ranking/unranking" begin
+    n = 5
+    k = 3
```

Comment:
could you also add some ranking/unranking tests for edgecases like `n = k = 0`, `n = k > 0`, `n > k = 1`? I just got bitten for not having a tests for such a case (x-ref https://github.com/oscar-system/Oscar.jl/pull/4878)